scripts/synthetix/facilitate_trade.js

```diff
@@ -60,8 +67,16 @@ module.exports = async callback => {
     const sUSDTosETHFee = parseFloat(snxjs.utils.formatEther(await snxjs.Exchanger.feeRateForExchange(sUSDKey, sETHKey)))
 
     // Compute buy-sell amounts based on unlimited orders with rates from above.
-    const [buyETHAmount, sellSUSDAmount] = calculateBuyAndSellAmountsFromPrice(formatedRate * (1 - sUSDTosETHFee), sUSD, sETH)
-    const [sellETHAmount, buySUSDAmount] = calculateBuyAndSellAmountsFromPrice(formatedRate * (1 + sETHTosUSDFee), sETH, sUSD)
+    const [buyETHAmount, sellSUSDAmount] = getUnlimitedOrderAmounts(
+      formatedRate * (1 - sUSDTosETHFee),
+      sUSD.decimals,
+      sETH.decimals
+    )
+    const [sellETHAmount, buySUSDAmount] = getUnlimitedOrderAmounts(
+      formatedRate * (1 + sETHTosUSDFee),
+      sETH.decimals,
+      sUSD.decimals
+    )
```

Comment:
`getUnlimitedOrderAmounts` has a different order input than `calculateBuyAndSellAmountsFromPrice`: the order of stable token and target token is swapped (see discussion in PR #134). Both tokens have 18 decimals, so it's not an urgent issue, but the decimals should be swapped  in both calls to avoid possible future issues.

Reply:
Thanks, this is what I wanted to know. Although I don't see any good reason for changing the order.